qa/tasks/mgr/dashboard/helper.py

```diff
@@ -514,6 +518,9 @@ def _validate_json(val, schema, path=[]):
     if isinstance(schema, JTuple):
         return all(_validate_json(val[i], typ, path + [i])
                    for i, typ in enumerate(schema.elem_typs))
+    if isinstance(schema, JUnion):
```

Comment:
This must look like
```
    if isinstance(schema, JUnion):
        for _, typ in enumerate(schema.elem_typs):
            try:
                if _validate_json(val, typ, path):
                    return True
            except _ValError:
                pass
        return False
```
because JLeaf will throw an error if the value doesn't match the specified type.

You may also add the doctest
```
>>> _validate_json({'num': 1}, JObj({'num': JUnion([int,float])}))
True

>>> _validate_json({'num': 'a'}, JObj({'num': JUnion([int,float])}))
False
```

Reply:
Thank you very much, @votdev ! Done!